custom_components/mojio/device_tracker.py

```diff
@@ -73,13 +77,51 @@ def login(self, hass):
+                car_attr = {"VIN": vehicle.getattribute("vin"),
+                            "name": vehicle.getattribute("name"),
+                            "parked": vehicle.getattribute("parked"),
+                            "status": vehicle.getattribute("status"),
+                            "direction": vehicle.getattribute("heading_direction"),
+                            "left_turn": vehicle.getattribute("left_turn"),
+                            "idle": vehicle.getattribute("idle"),
+                            "ignition_state": vehicle.getattribute("ignition_state"),
+                            "disturbance_state": vehicle.getattribute("disturbance_state"),
+                            "tow_state": vehicle.getattribute("tow_state"),
+                            "last_contact": vehicle.getattribute("last_contact"),
+                            "last_trip_id": vehicle.getattribute("last_trip_id"),
+                            "rpm": vehicle.getattribute("current_rpm", None),
+                            "speed_mph": vehicle.getattribute("current_speed_mph", None),
+                            "speed_kph": vehicle.getattribute("current_speed_kph", None),
+                            "seatbelt_warn": vehicle.getattribute("seatbelt_status_warning", False),
+                            "address": vehicle.location.getattribute("formatted_address"),
+                            "battery_voltage": vehicle.battery.getattribute("value"),
+                            "fuel_level": vehicle.fuel.getattribute("fuel_level", None),
+                            "virtual_fuel_level": vehicle.fuel.getattribute("virtual_fuel_level", None),
+                            "oil_temp_f": vehicle.engine_oil.getattribute("temp_f", None),
+                            "oil_temp_c": vehicle.engine_oil.getattribute("temp_c", None),
+                            "dtc_count": vehicle.dtc.getattribute("count"),
+                            "dtc_details": vehicle.dtc.getattribute("details", None),
+                            "lct_start": last_comp_trip.getattribute("start_date"),
+                            "lct_end": last_comp_trip.getattribute("end_date"),
+                            "lct_mpg": last_comp_trip.getattribute("mpg", None),
+                            "lct_kpl": last_comp_trip.getattribute("kpl", None),
+                            "lct_max_mph": last_comp_trip.getattribute("max_speed_mph", None),
+                            "lct_max_kph": last_comp_trip.getattribute("max_speed_kph", None),
+                            "lct_distance_mi": last_comp_trip.getattribute("distance_mi", None),
+                            "lct_distance_km": last_comp_trip.getattribute("distance_km", None),
+                            "lct_duration": last_comp_trip.getattribute("duration"), }
                 self._see(
-                    dev_id=vehicle.licence_plate.replace("-", "_"),
+                    dev_id=vehicle.id,
```

Comment:
Isn't `id` a UUID? I'd rather not have it as identifier for the entity. I think it's easier to work with the license plate e.g. when setting up automations.

What was the reason that you changed this to the ID?

Reply:
`id` is currently the license plate (with `-` or ` ` replaced) or VIN if no license plate (https://github.com/simontb/mojio-py/blob/master/mojio_sdk/vehicle.py#L22), and mojio_id is the UUID (https://github.com/simontb/mojio-py/blob/master/mojio_sdk/vehicle.py#L26)

Reply:
I added `mojio_id` because if you were using this outside of HA, you need that id to find trips that match that vehicle.  I added `id` because the tmobile app does not make you add a license plate so you could get null issues.

Reply:
Ah, I missed that. Thanks!